bedrock/settings/__init__.py

```diff
@@ -110,6 +110,7 @@
         "connect-src": list(set(_csp_default_src + _csp_connect_src)),
         # support older browsers (mainly Safari)
         "frame-src": _csp_child_src,
+        "upgrade-insecure-requests": True,
```

Comment:
Wondering if there is a good condition to skip this in local dev (which `if DEV` is not, as that is used in demo/dev deployments too…), or if this has to be kept unconditionally.